oap-mail/src/main/java/oap/mail/Transport.java

```diff
@@ -0,0 +1,5 @@
+package oap.mail;
+
+public interface Transport {
```

Comment:
Make Smtp transport implement this, making transport polymorphic thru the framework